vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -83,6 +83,7 @@ $global:map = @{
         "https://github.com/Microsoft/SDN/raw/master/Kubernetes/windows/hns.psm1",
         "https://globalcdn.nuget.org/packages/microsoft.applicationinsights.2.11.0.nupkg",
         "https://acs-mirror.azureedge.net/aks-engine/windows/provisioning/signedscripts-v0.0.14.zip",
+        "https://acs-mirror.azureedge.net/aks-engine/windows/provisioning/signedscripts-v0.0.15.zip",
```

Comment:
Please make sure this script is downloadable. I got BlobNotFound.

Reply:
Won't merge it if the build fails.

Reply:
The build passed.